docs/api/_templates/docs-navbar.html

```diff
@@ -37,6 +41,12 @@
         {%- include "search-field.html" %}
       {% endif %}
 
+      <div class="social">
+         <a class="social-icon" href="https://twitter.com/inrupt" target="_blank" title="Twitter"><i class="fab fa-twitter-square" ></i></a>
+         <a class="social-icon" href="https://www.linkedin.com/company/inrupt/" target="_blank" title="LinkedIn"><i class="fab fa-linkedin" ></i></a>
+         <a class="social-icon" href="https://forum.solidproject.org/" target="_blank" title="Solid Forum"> <i class="fas fa-users"></i></a>
```

Comment:
Readers who depend on a screen reader can not see what these links are. I think something like this would make it more accessible:

```suggestion
         <a class="social-icon" href="https://twitter.com/inrupt" target="_blank" title="Inrupt on Twitter" aria-label="Inrupt on Twitter"><i class="fab fa-twitter-square" aria-hidden="true"></i></a>
         <a class="social-icon" href="https://www.linkedin.com/company/inrupt/" target="_blank" title="Inrupt on LinkedIn" aria-label="Inrupt on LinkedIn"><i class="fab fa-linkedin" aria-hidden="true"></i></a>
         <a class="social-icon" href="https://forum.solidproject.org/" target="_blank" title="Solid Forum" aria-label="Solid forum"> <i class="fas fa-users" aria-hidden="true"></i></a>
```